context/app/static/js/components/entity-search/SearchWrapper/SearchWrapper.jsx

```diff
@@ -4,28 +4,33 @@ import Search from 'js/components/entity-search/Search';
 import { getDefaultFilters } from 'js/components/entity-search/searchkit-modifications/getDefaultFilters';
 import {
   mergeObjects,
-  getDonorMetadataFields,
+  buildDonorFields,
   createAffiliationFacet,
   createField,
   getEntityTypeFilter,
+  buildTileFields,
 } from './utils';
 import SearchConfigProvider from './provider';
 import { useNumericFacetsProps } from './hooks';
 
 function SearchWrapper({ uniqueFacets, uniqueFields, entityType }) {
-  const initialFacets = mergeObjects([
-    ...uniqueFacets,
-    ...getDonorMetadataFields(entityType),
+  const { tableFields: donorFacets } = buildDonorFields(entityType);
+
+  const initialFacets = Object.assign(
+    uniqueFacets,
```

Comment:
Is modifying `uniqueFacets` a desired side effect? If so, do we need `initialFacets` as well? If not, would spread be more clear?